mailpoet/lib/Config/Hooks.php

```diff
@@ -423,8 +423,12 @@ public function setupFooter() {
     );
   }
 
-  public function setFooter($text) {
-    return '<a href="https://feedback.mailpoet.com/" rel="noopener noreferrer" target="_blank">Give feedback</a>';
+  public function setFooter(): string {
```

Comment:
I don't think there's any functional difference, but if you're removing the unused `$text` you could update the `addFilter` call above to either omit the last `1` parameter or pass `0`. But `1` is the default anyway so it shouldn't matter at all.